### PR TITLE
Nations gamemode

### PR DESCRIPTION
**Pull request by tortellinitony**
:cl: Tortellini Tony
add: Nations gamemode ported from Paradise. Admin only
/:cl:

**TODO:**

1. [ ] Make nations verbs actually work
2. [ ] Make verbs into action buttons
3. [ ] Make heir, make leader, demote procs

### CODE REVIEW

**Comment by ghost** [comment]
i'm really not big into nations but i know others like it

i think some supporting code to help people run it more efficiently is better than just me saying "no i dislike the nations mode" but allowing people to run it anyway.

File: code/__DEFINES/atom_hud.dm [hunk 1]
```diff
@@ -40,6 +40,7 @@
 #define ANTAG_HUD_SINTOUCHED	16
 #define ANTAG_HUD_SOULLESS		17
 #define ANTAG_HUD_CLOCKWORK		18
+#define ANTAG_HUD_NATIONS		19
```
Comment (ghost) [inline code/__DEFINES/atom_hud.dm]: This can be done inside the `hippiestation` folder

File: code/modules/mob/mob.dm [hunk 2]
```diff
@@ -39,7 +39,7 @@
 /atom/proc/prepare_huds()
 	hud_list = list()
 	for(var/hud in hud_possible)
-		var/image/I = image('icons/mob/hud.dmi', src, "")
+		var/image/I = image('hippiestation/icons/mob/hud.dmi', src, "")
```
Comment (ghost) [inline code/modules/mob/mob.dm:42]: might just want to do a IF here to only pull our huds out of our folder

**Review by ghost — CHANGES_REQUESTED** [review]

**Comment by tortellinitony** [comment]
OK it should actually work now

File: code/modules/mob/mob.dm (hunk 2 above)
Comment (j-mie) [reply code/modules/mob/mob.dm:42]: I'm going to be putting a hook proc here for gang anyway, so he should put my hook in

**Comment by j-mie** [comment]
@tortellinitony You should wait for Gang to be merged, since I'll be adding a fuck tonn of hooks.

**Comment by tortellinitony** [comment]
So I've decided I'm not gonna take up adding war and peace initially, but I'll probably jam it in as a later update, same deal with nations chat. I'll get the verbs working, make them action buttons, and then this should be good to go

**Pull request closed without merging**